treadmill/appmgr/run.py

```diff
@@ -627,8 +626,8 @@ def _allocate_network_ports(host_ip, manifest):
 
     # Ephemeral port are the rest of the TCP ports
     manifest['ephemeral_ports'] = [
-        tcp_sock.getsockname()[1]
-        for tcp_sock in tcp_sock_iter
+        _tcp_sock.getsockname()[1]
```

Comment:
This would work without the `_` but `flake8` complains about reassigning the same variable again. We were creating `tcp_sock` again inside list comprehension.

There are a few places this kind of change is made due to similar reasons.